### ISSUE TITLE
Remove vue2-dropzone package

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#1953]
## Description
Remove vue2-dropzone package from files app since we don't really use it's features and what we use from it can be replaced with events.

**Comment by pascalwengerter** [comment]
@LukasHirt did you re-introduce vue2-dropzone in the evil PR (or decided to keep it, after all)?

**Comment by LukasHirt** [comment]
It was never removed, was it?

**Comment by pascalwengerter** [comment]
Nope. My understanding was that at some point in history it might've been possible (after #2239 was merged) but it seems to be a valid part of the application (again)? Then we can possibly close the issue

**Comment by LukasHirt** [comment]
When implementing the dropzone originally we used the mentioned lib. We are using only a super limited set of features that are easily replaced with native events so we do not need that lib. This ticket is not about removing the dropzone itself. Only the lib under the hood should be removed and replaced with those native events...

**Comment by pascalwengerter** [comment]
I see, thanks for clarification!

**Comment by pascalwengerter** [comment]
Closing in favor of #6019
